docs/Doxyfile

```diff
@@ -2306,16 +2273,16 @@ DIA_PATH               =
 # and usage relations if the target is undocumented or is not a class.
 # The default value is: YES.
 
-HIDE_UNDOC_RELATIONS   = YES
+HIDE_UNDOC_RELATIONS   = FALSE
```

Comment:
Why this change?  And why FALSE instead of NO?

Reply:
`FALSE` is a mistake obviously.
This change is a leftover of me trying to find the right parameters for Doxygen to generate appropriate call/caller graphs.
I changed it back to `NO`